source/_posts/2023-08-01-php-core-roundup-15.md

```diff
@@ -0,0 +1,366 @@
+---
+title: 'PHP Core Roundup #15'
+layout: post
+tags:
+  - roundup
+author:
+  - name: Ayesh Karunaratne
+    url: https://aye.sh
+  
+  - name: Sergey Panteleev
+    url: https://sergeypanteleev.com
+published_at: 01 August 2023
+
+---
+
+// Todo add intro.
+
+> The PHP Foundation is a collective of PHP contributors, veterans, and companies that collaborate to ensure the long-term sustainability of the PHP programming language. The foundation currently supports six contributors focused on PHP's maintenance, debt reduction, and feature development. These contributors work closely with others on coding, documentation, and discussions.
+
+<br>
+
+> Started over a year ago, the PHP Core Roundup series offers a summary of the latest developments, discussions, and news about PHP Core, contributed by both PHP Foundation members and other participants. This post is the thirteenth in the PHP Core Roundup series.
+
+{% include "newsletter.html" %}
+
+// Todo: Add PHP beta releases
+
+## Releases
+
+// Todo Add releases.
+
+
```

Comment:
```suggestion
## PHP 8.3 Feature-Freeze

On July 18, PHP 8.3 branch reached its feature-freeze. As the name suggests, the list of features we plan to ship with PHP 8.3 is now frozen. Contributors who wish to make substantial changes to PHP should now target the next PHP version, PHP 8.4.

In the coming weeks, the PHP Foundation members, the PHP development team, and contributors will be making improvements to get PHP 8.3 ready for production release.

Release managers elected for PHP 8.3, Pierrick Charron, Jakub Zelenka, and Eric Mann will have the final say in case a major change must be made to the PHP 8.3.

## PHP 8.3 Beta Release

The second beta release of PHP 8.3 was released this week. Now would be an ideal time to test your PHP applications on PHP 8.3.

Compiled Windows binaries are available at [windows.php.net/qa](https://windows.php.net/qa/), [Docker images](https://hub.docker.com/_/php/tags?page=1&name=8.3.0) are available at Docker Hub, and source code at [php/php-src repository on GitHub](https://github.com/php/php-src) to compile yourself. On Homebrew, `PHP 8.3.0-dev` packages are available from <code>[shivammathur/php](https://github.com/shivammathur/homebrew-php)</code> tap.

CI/CD platforms that use Docker images can use the PHP 8.2 docker images available with various base images. GitHub Actions can also make use of <code>[shivammathur/setup-php](https://github.com/shivammathur/setup-php)</code> action, which supports PHP 8.3 builds.

## Releases

The PHP development team released two new versions in July 2023:

**[PHP 8.2.8](https://www.php.net/archive/2023.php#2023-07-06-2)** and **[PHP 8.1.21](https://www.php.net/archive/2023.php#2023-07-06-3)**

These releases include bug fixes across various components such as CLI, Core, Curl, DOM, Opcache, OpenSSL, PGSQL, Phar, SPL, and Standard.
```